app/services/fast_track/hypertension_pdf_generator.rb

```diff
@@ -171,72 +125,78 @@ def add_medications_intro(pdf)
+    def add_about
+      @pdf.text  "\n"
+      @pdf.text  'About this Document', size: 14
+      ABOUT_LINES.each do |line|
+        @pdf.text line, size: 11, inline_format: true
       end
 
-      pdf.table(med_rows, cell_style: { size: 8, inline_format: true })
-
-      pdf
-    end
-
-    def about_lines
-      ['The Hypertension Rapid Ready for Decision system retrieves and summarizes ' \
-       'VHA medical records related to hypertension claims for increase submitted on va.gov. ' \
-       'VSRs and RVSRs can develop and rate this claim without ordering an exam if there is '\
-       'sufficient existing evidence to show predominance according to ' \
-       '<link href="https://www.ecfr.gov/current/title-38/part-4">' \
-       '<color rgb="0000ff">DC 7101 (Hypertension) Rating Criteria</color></link>. ' \
-       'This is not new guidance, but rather a way to ' \
-       '<link href="https://www.ecfr.gov/current/title-38/chapter-I/part-3/' \
-       'subpart-A/subject-group-ECFR7629a1b1e9bf6f8/section-3.159"><color rgb="0000ff">' \
-       'operationalize existing statutory rules</color><link> in 38 U.S.C § 5103a(d).',
-       "\n",
-       'Not included in this document:',
-       ' •  Private medical records',
-       ' •  VAMC data for clinics using CERNER Electronic Health Record system ' \
-       '(Replacing VistA, but currently only used at Mann-Grandstaff VA Medical Center in Spokane, Washington)',
-       ' •  JLV/Department of Defense medical records'].freeze
-    end
-
-    def add_about(pdf)
-      pdf.text  "\n"
-      pdf.text  'About this Document', size: 14
-      about_lines.each do |line|
-        pdf.text line, size: 11, inline_format: true
-      end
+      @pdf
+    end
+
+    ABOUT_LINES = [
+      'The Hypertension Rapid Ready for Decision system retrieves and summarizes ' \
+      'VHA medical records related to hypertension claims for increase submitted on va.gov. ' \
+      'VSRs and RVSRs can develop and rate this claim without ordering an exam if there is '\
+      'sufficient existing evidence to show predominance according to ' \
+      '<link href="https://www.ecfr.gov/current/title-38/part-4">' \
+      '<color rgb="0000ff">DC 7101 (Hypertension) Rating Criteria</color></link>. ' \
+      'This is not new guidance, but rather a way to ' \
+      '<link href="https://www.ecfr.gov/current/title-38/chapter-I/part-3/' \
+      'subpart-A/subject-group-ECFR7629a1b1e9bf6f8/section-3.159"><color rgb="0000ff">' \
+      'operationalize existing statutory rules</color><link> in 38 U.S.C § 5103a(d).',
+      "\n",
+      'Not included in this document:',
+      ' •  Private medical records',
+      ' •  VAMC data for clinics using CERNER Electronic Health Record system ' \
+      '(Replacing VistA, but currently only used at Mann-Grandstaff VA Medical Center in Spokane, Washington)',
+      ' •  JLV/Department of Defense medical records'
+    ].freeze
```

Comment:
You could get rid of quotes and backslashes using [heredoc syntax](https://riptutorial.com/ruby/example/2933/multiline-strings) for multiline strings.

Reply:
I'll give that a shot and see if it helps with readability. Thanks!

Reply:
> You could get rid of quotes and backslashes using [heredoc syntax](https://riptutorial.com/ruby/example/2933/multiline-strings) for multiline strings.

I'm very interested in makign this easier to work with. It looks like you have to do some [weird stuff](https://www.ruby-forum.com/t/array-of-here-documents/128579/5) to get it to work in this specific context:

```ruby
stuff = [ <<DOC1, <<DOC2, <<DOC3 ]
Hello World
DOC1
How’s it going?
DOC2
It’s just crazy, thanks.
DOC3

p stuff
#=> [“Hello World\n”, “How’s it going?\n”, “It’s just crazy, thanks.
\n”]
```

I'll toy around some more but it would be nice to remove this mix of string concatenation